userguide/jails.rst

```diff
@@ -624,6 +624,12 @@ Click :guilabel:`NEXT` to view all jail
    |                        |              | addresses, enter the host MAC address and the jail MAC address separated by a space.                    |
    |                        |              |                                                                                                         |
    +------------------------+--------------+---------------------------------------------------------------------------------------------------------+
+   | nat_forwards           | checkbox     | `Network Address Translation (NAT) port forwarding <https://en.wikipedia.org/wiki/Port_forwarding>`__   |
```

Comment:
Shouldn't this be _NAT Forwards_, or are we still trying to stay with the same names as iocage fields?

Reply:
This is what it is in the UI right now. I don't doc what it isn't